packages/frontend-2/components/settings/workspaces/security/sso/Wrapper.vue

```diff
@@ -16,7 +16,7 @@
         </div>
         <FormButton
           v-if="workspace.hasAccessToSSO"
-          :disabled="isFormVisible || !!provider"
+          :disabled="!isWorkspaceAdmin || isFormVisible || !!provider"
```

Comment:
Probably should add a tooltip here, something like "Ask your workspace admin to enable SSO" or something like that